src/site/content/en/blog/compute-pressure/index.md

```diff
@@ -0,0 +1,284 @@
+utilization of the browser's device. The API is designed as a callback mechanism
+that receives periodic updates, allowing developers to observe changes in CPU
+utilization over time. A web app that uses this API can make informed decisions
+about enabling or disabling heavyweight features in response to these
+observations. It's a feedback mechanism that unlocks new avenues for adaptive
+behavior, which can in turn lead to smoother, cooler experiences across a range
+of hardware devices.
+
+## Use cases
+
+The motivating use cases are helping video conferencing and video game web
+applications adapt to CPU utilization. The initial experimental implementation
+of the API aims to support the following potential decisions:
+
+### Video conferencing
+
+- Dynamically adjust the number of live video feeds displayed on the screen at
+  once.
+- Adjust the video resolution or frame rate.
+- Skip non-essential video processing, such as cosmetic filters.
+- Disable non-essential audio processing, such as noise suppression.
+- Skew the quality-vs-speed and size-vs-speed knobs towards "speed" in video and
+  audio encoding.
+
+### Video games
+
+- Use lower-quality video and audio assets as needed.
+- Disable optional effects like reflections, glare, or particle animations that
+  don't affect gameplay.
+- Tweak quality-vs-speed knobs in the game's rendering engine, affecting areas
+  like shadows quality, texture filtering, and view distance.
+
+The secondary use case is measuring the CPU resource consumption of a feature.
+This ultimately supports the main goal of avoiding driving user devices into
+high CPU utilization. The API aims to help developers make the following types
+of decisions:
+
+- Compare the CPU consumption of alternative implementations of the same
+  feature, to find out which is more efficient. The API supports measuring CPU
+  utilization in the field via
+  [A/B tests](https://en.wikipedia.org/wiki/A/B_testing), because a feature's
+  CPU utilization depends on the hardware it's running on. Most developers
+  cannot afford performance measurement labs covering all the devices owned by
+  their users.
+- Estimate the impact of enabling a feature on CPU consumption. This cost
+  estimate feeds into the decisions outlined in the primary use case.
+
+## Core concepts
+
+### CPU utilization
+
+The CPU utilization of a device is the average of the utilization of all the
+device's [CPU cores](https://en.wikipedia.org/wiki/Multi-core_processor),
+represented in the API as a number between `0.0` and `1.0`.
+
+A CPU core's utilization is the fraction of time that the core has been
+executing code belonging to a thread, as opposed to being in an idle state.
+
+A CPU utilization close to `0.0` indicates that the processor has been almost
+entirely idle. A CPU utilization close to `1.0` is very likely to lead to a
+device that is generating excessive heat. Web applications can help avoid bad
```

Comment:
```suggestion
entirely idle. A CPU utilization close to `1.0` is very likely to
generate excessive heat. Web applications can help avoiding bad
```

Reply:
>help avoiding

I've been failing to mark situations like this. "Avoiding" is the start of the object portion of the sentence. (_subject verb_ [_object_]). Consequently, it must be a noun. A verb functioning as a noun must be a gerund and therefore a present participle verb. I don't know if you followed that, but I'm providing in case you want to dig into the logic.